contributors/devel/cherry-picks.md

```diff
@@ -9,7 +9,7 @@ depending on the point in the release cycle.
 1. Cherrypicks are [managed with labels and milestones](pull-requests.md#write-release-notes-if-needed)
 1. To get a PR merged to the release branch, first ensure the following labels
    are on the original **master** branch PR:
-  * An appropriate milestone (e.g. v1.3)
+  * The `milestone` for the branch that you want to cherry-pick to (e.g. v1.7 if you want to backport your change in 1.7 branch)
```

Comment:
Style nit: milestone is like "label"... it's an object type, as opposed to the name of a specific object. It should probably have the single backticks removed, while `v1.7` in the example would be good to single backtick. LGTM otherwise :)

Reply:
Sounds good. I've removed the backticks.